### PR TITLE
Adding Vivid Pool

### PR DESCRIPTION
**Pull request by rere-dnaw**
added Vivid Pool

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3656,5 +3656,19 @@
     "telegram": "hungbv2",
     "operatorName": "Bui Viet Hung",
     "github": "sirhung1993"
+  },
+  {
+    "ticker": "VIP",
+    "poolId": "pool1r9zrp0hpy3dj6lse5vl9yc672v5w7fzrrp62pjcersqe2djm409",
```
Comment (Bez625) [inline registry.json:3662]: This is incorrect - i think your pool id is actually 194430bee1245b2d7e19a33e52635e5328ef24431874a0cb191c0195. Please correct

File: registry.json [hunk 2]
```diff
@@ -3656,5 +3656,19 @@
...
+    "social": {
+      "website": "https://vivid-pool.info",
+      "twitter": "https://twitter.com/VividPool",
```
Comment (Bez625) [inline registry.json:3665]: Please remove the leading https://twitter.com/

File: registry.json [hunk 3]
```diff
@@ -3656,5 +3656,19 @@
...
+      "facebook": "",
+      "instagram": "",
+      "youtube": "https://www.youtube.com/channel/UC8vR0zHl2l9AlGPiWKOxDpA"
+    },
+    "telegram": "vivid_pool",
```
Comment (Bez625) [inline registry.json:3670]: Please make sure this is TG handle of a user and not a group or channel. This looks like a channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 3 above)
Comment (rere-dnaw) [reply registry.json]: TG handle changed

File: registry.json (hunk 2 above)
Comment (rere-dnaw) [reply registry.json]: Removed

File: registry.json (hunk 1 above)
Comment (rere-dnaw) [reply registry.json]: The pool Id updated

**Review by Bez625 — APPROVED** [review]

**Pull request merged**